src/components/NavBar.vue

```diff
@@ -241,14 +239,63 @@ export default {
           </v-icon>
         </v-btn>
       </template>
-      <v-card tile class="pa-0" max-width="320">
+      <v-card tile class="pa-0" max-width="500">
         <v-card-text class="pb-0">
-          <p>
+          <div class="mb-2">
             <span v-if="connected">Connected</span>
             <span v-else-if="connecting">Connecting</span>
             <span v-else>Couldn't connect</span>
-            to <span class="font-weight-bold">{{ url }}</span> <br /><br />
-          </p>
+            to
+            <span class="font-weight-bold">
+              <span v-if="isCloud" class="primary--text">Prefect Cloud</span>
+              <span v-else class="secondaryGray--text">Prefect Server</span>
+            </span>
+            <span v-if="isServer">
+              at
+              <span class="font-weight-bold">{{ url }}</span>
+            </span>
+
+            <v-alert
+              v-if="isServer && (!connected || connecting)"
+              border="left"
+              colored-border
+              class="text-body-1 mt-2"
+              icon="cloud"
+              color="primary"
+              tile
+            >
+              <div>
+                Having trouble?
+                <span class="font-weight-medium">Don't panic! </span>
+              </div>
+              <div class="mt-4">
+                The <v-icon x-small>fab fa-slack</v-icon>&nbsp;
+                <ExternalLink
+                  href="https://join.slack.com/t/prefect-community/shared_invite/enQtODQ3MTA2MjI4OTgyLTliYjEyYzljNTc2OThlMDE4YmViYzk3NDU4Y2EzMWZiODM0NmU3NjM0NjIyNWY0MGIxOGQzODMxNDMxYWYyOTE"
+                  >Prefect Slack community</ExternalLink
+                >
```

Comment:
Should we also include a link to github discussions here?  There may already be a discussion around their issue?

Reply:
I added a link :)